### ISSUE TITLE
Revise Extended Data and SI to journal style

### ISSUE THREAD

**Issue by ahwagner** [ahwagner/vicckb#20]
> Please revise the Extended Data and Supplementary Information callouts to match journal style (see attached “NRG Formatting Guide_v.6” and “Inventory of Supporting Information” for detailed instructions on formatting). In particular, please reformat “Supplementary Figures 1-6” as Extended Data Figures 1-6, present the “Supplementary Discussion” and “Supplementary Data” as a single Supplementary Note with subheadings, and include the name of each Supplementary Table (e.g. Supplementary Table 1, etc.) alongside the corresponding title at the top of each sheet in the Excel file.

**Comment by ahwagner** [comment]
[Inventory_of_Supporting_Information_LS_v.8.2019_1571679324_7.docx](https://github.com/ahwagner/vicckb/files/3893874/Inventory_of_Supporting_Information_LS_v.8.2019_1571679324_7.docx)
 
[NRG_Formatting_Guide_v.6.August_1571679324_2.docx](https://github.com/ahwagner/vicckb/files/3893877/NRG_Formatting_Guide_v.6.August_1571679324_2.docx)

**Comment by ahwagner** [comment]
To manage this (and related issues for references, etc.) first combining into a single document:
https://docs.google.com/document/d/1CUSdFIaGBdWMyyfXAGctcSjZuoKHRI4m/edit#heading=h.gbu1e9pw1bjv

**Comment by ahwagner** [comment]
We have revised the extended data figure callouts to match journal style. We combined the supplementary discussion and data into a single supplementary note along with the above-requested VICC members. We have revised the supplementary tables to indicate the table name/number alongside the title for each sheet.

Documents to submit:
- [ ] Supplementary Information
- [ ] Inventory of Supporting Information
- [ ] Nature Research's Reporting Summary
- [ ] Supplementary Tables
